legal-api/tests/unit/models/test_business.py

```diff
@@ -204,6 +204,38 @@ def test_business_find_by_identifier_no_identifier(session):
     assert b is None
 
 
+def test_good_standing(session):
```

Comment:
Please turn this into 1 test with parameters for the current time and consider using the ```freeze``` package.
eg https://github.com/bcgov/lear/blob/master/legal-api/tests/unit/api/test_business_comments.py#L97

Reply:
This is now parameterized. I can look at adding the freeze if you think it'd be valuable but not sure it is?